packages/edit-site/src/components/header/fullscreen-mode-close/index.js

```diff
@@ -21,10 +20,12 @@ function FullscreenModeClose() {
 		return null;
 	}
 
+	const buttonIcon = ! isEmpty( icon ) ? icon : wordpress;
```

Comment:
```suggestion
	const buttonIcon = icon || wordpress;
```
When would an icon ever be falsy?

Reply:
Updated in cbc8eba28bdb0ae229cbc7a54c0e4190eff813e9.